tests/test_wsireader.py

```diff
@@ -1078,7 +1093,7 @@ def test_virtual_wsi_reader_invalid_mode(source_image):
 
 
 def test_virtual_wsi_reader_read_bounds(source_image):
```

Comment:
We should split this into smaller test. It is trying to do too many things in one.

Reply:
In a separate PR I would suggest.